### DISCUSSION TITLE
Set an expiration on the shared link

### DISCUSSION THREAD

**Discussion by metmarkosaric** [plausible/analytics#2072] [Anything Else]
"Generating private, unique URLs are a decent alternative to proper authentication, but to improve security, I’d like to be able to generate these unique URLs on-demand and delete them after an hour or so. The API has a way to create new unique URLs, but not delete them. Are there plans to enhance the API to allow for that, or to allow setting an expiration on the links (preferable)."

(feedback via email)

**Comment by bkosborne** [comment]
This was my request :) thanks for adding it.
